message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -241,6 +249,36 @@
         <param index="6">Reserved, send 0</param>
         <param index="7">Reserved, send 0</param>
       </entry>
+      <entry value="43000" name="MAV_CMD_GUIDED_CHANGE_SPEED">
+        <description>Change flight speed at a given rate. This slews the vehicle at a controllable rate between it's previous speed and the new one. (affects GUIDED only. Outside GUIDED, aircraft ignores these commands. Designed for onboard companion-computer command-and-control, not normally operator/GCS control.)</description>
+        <param index="1" enum="SPEED_TYPE" label="speed type">Type (see SPEED_TYPE enum), airspeed or groundspeed.</param>
```

Comment:
No need for duplicated information. The units should not be present in the description text, only in the units attribute